Runtime/Behaviours/CinemachineBrain.cs

```diff
@@ -873,7 +873,7 @@ private void PushStateToUnityCamera(in CameraState state)
                     cam.lensShift = state.Lens.LensShift;
                     cam.orthographic = state.Lens.Orthographic;
                     cam.usePhysicalProperties = state.Lens.IsPhysicalCamera;
-                    if (state.Lens.IsPhysicalCamera)
+                    if (state.Lens.IsPhysicalCamera && state.Lens.ModeOverride == LensSettings.OverrideModes.Physical)
```

Comment:
I don't think this is the right fix.  What if it's a physical camera with no override and the vcam wants to set the gate fit?  I think the real problem is that the gate fit (which is newly added in the vcam) doesn't get properly initialized and defaults to "None".  It's supposed to drive the camera (like the clip planes), but we need to somehow handle the case where it's uninitialized, and pull the value from the camera - otherwise we always push it

Reply:
I looked more closely, and changed my mind about this.  It seems that the GateFit is only settable n the inspector if Mode Override is physical, so this fix is good after all